core/controllers/feedback.py

```diff
@@ -158,7 +158,7 @@ class SuggestionActionHandler(base.BaseHandler):
     _ACCEPT_ACTION = 'accept'
     _REJECT_ACTION = 'reject'
 
-    @editor.require_editor
+    @acl_decorators.can_manage_suggestions_on_exploration
```

Comment:
I see. Two ways: you could change the URL to include state name, or (better, I think) you could change the other URL(s) to exclude state name and pass the latter in as a parameter instead so that all handler methods have a consistent signature.